src/Shared/AzureManaged/DurableTaskSchedulerOptions.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+using Azure.Core;
+using Azure.Identity;
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Options for configuring the Durable Task Scheduler.
+/// </summary>
+public class DurableTaskSchedulerOptions
```

Comment:
Typically we don't want public types in shared files, as then users get conflicting copies of this type when they import the two different packages.

We may need a `Microsoft.DurableTask.Abstractions.AzureManaged` package to unify these public types.

Reply:
as discussed, will split this in client/worker